inc/class-registration.php

```diff
@@ -492,6 +492,18 @@ public function enqueue_dependencies( $post = null ) {
 		if ( ! self::$scripts_loaded['product-image'] && has_block( 'themeisle-blocks/product-image', $post ) ) {
 			wp_enqueue_script( 'wc-single-product' );
 		}
+
+		// DEBUG
+		// TODO: load this only when a block has a sticky block.
+		$asset_file = include OTTER_BLOCKS_PATH . '/build/blocks/sticky.asset.php';
+		wp_enqueue_script(
```

Comment:
Any idea on how to load this only if sticky is present? Should we analyze the `className` attr?